app/views/admin/feedback_messages/_style.html.erb

```diff
@@ -22,4 +22,21 @@
     color: black;
     text-align: center;
   }
+  .badge-connect {
```

Comment:
Same here, do our pre-commit hooks format our css?

Reply:
I don't think it works in CSS blocks inside ERB files. Only in (S)CSS standalone files

Reply:
What @rhymes said @katiedavis.